fotoapparat/src/test/java/io/fotoapparat/parameter/provider/InitialParametersProviderTest.java

```diff
@@ -129,4 +129,16 @@ public void selectPreviewSize_WithValidAspectRatio() throws Exception {
         );
     }
 
+    @Test(expected = IllegalArgumentException.class)
+    public void initialParameterMissing() throws Exception {
```

Comment:
This test tests validator and not provider. I suggest we make validator non-static and then just verify that it was called